tsconfig.json

```diff
@@ -7,9 +7,8 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
     "noImplicitReturns": true,
-    "allowJs": true,
     "outFile": "src/main/resources/typescriptc.js"
```

Comment:
If we move the resources folder outside of src it could do a trick, what do you think?